### PR TITLE
Add access.conf template

### PR DESCRIPTION
**Pull request by jruijgers**
Add access.conf template
Render the template

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@
       run_opts: ''
       playbook: travis_test.yml
 
+
```
Comment (vovinacci) [inline .travis.yml]: Remove blank line here

File: README.md [hunk 2]
```diff
@@ -10,9 +10,14 @@ Usage
...
 ```YAML
-  ### TODO: Place configuration example here
+users:
+  sample_user:
+    key_base64: "YOUR_GENERATED_KEY"
+    hmac_base64: "YOUR_GENERATED_HMCA_KEY"
+    fw_access_timeout: 30 # OPTIONAL
 ```
 
+
```
Comment (vovinacci) [inline README.md]: Remove blank line here

File: tests/travis_test.yml [hunk 3]
```diff
@@ -6,3 +6,9 @@
         role: "role_under_test",
         docker_test: true
       }
+
+  vars:
+    users:
+      test:
+        key_base64: "test"
+        hmac_base64: "hmac"
```
Comment (vovinacci) [inline tests/travis_test.yml]: Do we use those vars?

**Review by vovinacci — CHANGES_REQUESTED** [review]
Almost there :)

File: tests/travis_test.yml (hunk 3 above)
Comment (jruijgers) [reply tests/travis_test.yml]: Yes, we use those vars in the access.conf

**Review by vovinacci — APPROVED** [review]

**Pull request merged**